board/piksiv3/rootfs-overlay/etc/init.d/S86skylark1_upload_daemon

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
Are you using `skylark1` and `skylark0` to control ordering? That should definitely be done with `S8x` (or move to `S9x`) instead.